### ISSUE TITLE
No example .env.example in distro

### ISSUE THREAD

**Issue by breckbaldwin** [rok-cesnovar/cloudstan-experiment#4]
No luck finding anything in the repo per instructions
find . -name ".env.example"

Hope I am not missing something stupid. Got a class coming up and want to use, figure it is better to administer myself.

thanks
Breck

**Comment by rok-cesnovar** [comment]
Hi Breck,

yeah it seems to be missing. I will add it. Thanks for the report.

If you want you can just use the one already running at https://cloudstan-experiment.herokuapp.com/
The webpage works, but if no one uses it, the instance goes to sleep and the reaction time for the first access is a few seconds. Once it starts its all fine.

I can assign it a more powerful AWS instance for the workshop if you want. Send me an e-mail with info on how many participants you will have and when is it happening so I can arange that.

I also need to update it with the latest update for the visualizations, I made them a bit more performant. Will do that tomorrow.

I stopped developing this because Erik Štrumbelj is working on a completely new frontend and we are also redesigning the cloud backend. So turns out that this one was a proof of concept for what is to come later this month hopefully :)

**Comment by breckbaldwin** [comment]
Don't see the .env.example still. 

I tried again, BreckTest2, still get compile errors, works with CmdStan2.20. 

Given that this is a proof of concept I don't really want to bother you with this. If you can get me the .env.example file I'll try and sort things out myself. The hardest part of cloud dev is getting stuff configured so an example config appreciated. 


Breck


Code if you want to deal with the bug is:

parameters {
	   real <lower=1,upper=5> chance_in_5;
	   //real chance_in_5;
}

transformed parameters {
	    real LP = target();
}

model {

 print("chance_in_5: ",chance_in_5);
 print("target start on log scale: ",target()," decimal scale: ", exp(target()));
  print("");
  {
  real foo = 10;
   }
}

generated quantities {
	  print(LP);
}

**Comment by rok-cesnovar** [comment]
I think the backend just crashed that is why this didnt compile. I am upgrading httpstan right now. For the new non-prototype we are building we are relying directly on cmdstan (via a cmdstan.js wrapper) to avoid any other interface related issues and httpstan is still very much in development.

I wasnt actively monitoring what is going on here because I didnt think anyone was actually using this. Sorry I dropped the ball.

As for the the .env.example it should look like [this](https://github.com/madhums/node-express-mongoose/blob/master/.env.example)

With the values for MONGODB_URL and HTTPSTAN_URL set. The first one is the URL to the Mogodb database (I set up a free Mongodb Atlas DB, takes you 3 minutes).

**Comment by rok-cesnovar** [comment]
Yeah, I reinstalled httpstan and I was able to compile the model. Let me know if I can help any further.
